requirements.txt

```diff
@@ -2,7 +2,7 @@ celery[redis]
 pyoai
 PyPDF2
 bibtexparser>=1.1.0
-redis<3
+redis
```

Comment:
This one is tricky, I got an error that version 3.2.0 was required. This is coming from https://github.com/celery/kombu/pull/1011 (which is pulled as a dependency to another lib, maybe `celery[redis]`?).

@wetneb You forced a downgrade to `redis<3` a while ago (https://github.com/dissemin/dissemin/commit/cafc862da005ea25c816018d5d116f1d538d5f89), do you remember why? My change might be problematic.

Reply:
That's apparently because celery 4.3.0 was released. I think the previous version was not compatible with redis 3. So removing this should be fine now!